## Biotech 7005: Frontiers of Biotechnology: Bioinformatics and Systems Modelling

Semester 2 2017

Coordinator:
- Dan Kortschak

Instructors:
- Jimmy Breen
- Stephen Pederson
- Dan Kortschak

__*Course Contact*__
Dan Kortschak
Email: dan.kortschak@adelaide.edu.au
Phone: 08 8313 0928
Office: Braggs 2.27

## Contact Hours

Lectures: Friday 9am, Schulz, 308
- 27 Jul — 14 Sep
- 5 Oct — 26 Oct

Tutorials: Friday 9am-11pm, Mawson, 134, Sprigg Room
- 28 Jul — 15 Sep
- 6 Oct — 27 Oct

Practicals: Monday 1pm-3pm, Mawson, G12 Computer Suite
- 24 Jul — 11 Sep
- 9 Oct — 23 Oct

## Timetable

| **Week** | **Monday** |**Lecture**    | **Practical** |
|----------|------------|---------------|---------------|
| **1** | 24/7 | Introduction to Bioinformatics (Dan)| Introduction to R (Steve) (1) |
| **2** | 31/7 | Statistics (Steve) | Introduction to R (Steve) (2) |
| **3** | 7/8  | Evolutionary Processes (Dan) | Command-line (Steve) (1) |
| **4** | 14/8 | Alignment Process (Dan) | Command-line (Steve) (2) |
| **5** | 21/8 | Alignment Process (Dan) | Evolutionary processes (Dan) (1) |
| **6** | 28/8 | Next-generation sequencing (Dan) | NGS data practical (Steve/Jimmy) (1) |
| **7** | 4/9  | Genomics/Repeats/Metagenomics (Jimmy) | NGS data practical (Steve/Jimmy) (2) |
| **8** | 11/9 | Non-coding RNA/Transcriptomics (Steve) | RNAseq practical (Steve) (1) |
| **-** | 18/9 | Semester Break |
| **-** | 25/9 | Semester Break |
| **9** | 2/10 (PH) | Epigenetics (Jimmy) | No practical (PH) |
| **10** | 9/10 | Introduction to Systems Biology (Dan)  | [Graphical analyses (Dan) (1)](https://github.com/kortschak/graphprac)  |
| **11** | 16/10 | Systems Biology (Ville) | Systems Biology practical (Ville) (1) |
| **12** | 23/10 | The Future — wooooooo (Dan) | Wrap-up (Jimmy/Dan/Steve)  |


# Assessment Schedule

| **Assessment** | **Subject** | **Weight** | **Due Date** |
|----------------|-------------|------------|--------------|
| Assessment 1 | R programming | 7% | Friday 4th August |
| Assessment 2 | Command-line | 7% | Friday 18th August |
| Assessment 3 | Evolutionary processes | 7% | Friday 25th August |
| Assessment 4 | NGS data practical | 7% | Friday 8th September |
| Assessment 5 | RNAseq practical | 7% | Friday 15th September |
| Assessment 6 | Graphical analyses | 7% | Friday 13th October |
| Project | Complete Dataset | 18% | Friday 20th October |
| Exam | | 40% | |

### Assessment Checklist

Have you:

- [ ] Answered all the questions?
- [ ] Followed naming conventions for Assessments?
- [ ] Attached an assessment coversheet?
	- This is a course requirement. See the [Academic Honesty Policy](http://www.adelaide.edu.au/policies/230/).
- [ ] Identified the work as yours?
	- Emails should have the course and assessment task names.
	- Documents should be named with your name, the course name and the assessment task.
	- Printed documents should have you name and the course and assessment task in the text/footer/header.
- [ ] Used appropriate electronic communication with assessors?
	- Emails should have a meaningful subject.
- [ ] Handed in the assignment before the due time (Friday tutorial 9am)?

#### Major Project (15%)

In this course, the following next-generation sequencing (NGS) datasets/protocols will be in detail:

1. Whole genome sequencing/Resequencing
2. Transcriptome Sequencing (RNAseq)
3. DNA Methylation/Bisulfite Sequencing
4. Enrichment/Capture sequencing (Methyl-capture, ChIPseq, RIPseq)
5. Metagenomics/Microbial profiling

Each of these NGS approaches uses similar programs and analysis approaches, such as quality control (quality and sequencing adapter trimming), genome alignment, and downstream visualisation and statistical methods. For the major project, you will take a published dataset and complete all the analysis tasks (from raw data to final results) during the course.

You have the freedom to choose any dataset you would like before we start week 5's practical (NGS data practical), or choose from four datasets that we have chosen. You will be asked assessment questions on your data during practical assessments throughout the semester, and a final report will be due the week before the end of semester (Week 12)


#### Practical Assignment 1 - Introduction to command-line (9%)

Week 1-2

__Question 1__

Submit the relevant parts of your terminal history (copy-pasting will suffice) for the following commands:

It should be possible for another person to completely reproduce your output from the history you submit.

__Question 2__

At the start of the practical you were told that the following practices should be adopted:

- Do not use spaces in filenames.
- Extensively comment your code with meaningful comments.
- Use robust error handling.
- Report version numbers of programs used.
- Use sensible variable names.
- File checks: Checking if your code is valid and runs successfully.

In a long-format question, explain why  these practices are important?

The following articles will help you understand the importance of some of these.
Others may take some thought, discussion with other students or careful questions.

[How To Ask Questions The Smart Way](http://www.catb.org/esr/faqs/smart-questions.html)

[How to write a good bug report](https://musescore.org/en/developers-handbook/how-write-good-bug-report-step-step-instructions)

#### Practical Assignment 2 - Introduction to command-line scripting (9%)

Week 3-4

__Question 1__



__Question 2__


#### Practical Assignment 3 - Next-Generation Sequencing (9%)

Week 5-6

__Question 1__



__Question 2__


#### Practical Assignment 4 - R programming (9%)

Week 7-8

__Question 1__



__Question 2__


#### Practical Assignment 5 - Systems Biology (9%)

Week 9

__Question 1__



__Question 2__


#### Practical Assignment 6 - Metagenomics (9%)

Week 10-11

__Question 1__



__Question 2__
